modules/verifybamid2/main.nf

```diff
@@ -16,11 +15,13 @@ process verifybamid2 {
 
     script:
     def reference = ref_fasta ? /--Reference "${ref_fasta}"/ : ''
+
+    vf_threads = (int) Math.ceil(task.cpus*2)
```

Comment:
This is a lot of code for just *2 a variable... I dont think it is needed

Reply:
suggestion updated and tested